README.md

```diff
@@ -1,2 +1,96 @@
 # d2-logger
-DHIS2 library that allows a certain application to register events in a DHIS2 program
+
+DHIS2 library that allows a certain application to register logs as events in a DHIS2 program or display logs on the console.
+
+## Usage
+
+There are two types of logger output:
+
+1. Using a DHIS2 events program to register the logs:
+
+    You will need to have and existing event program in DHIS2 with the following data elements with value type Text: Message and MessageType. MessageType would be and option set with the following options: "Error", "Warn", "Success", "Info" and "Debug".
+
+    Therefore, the following configuration will be passed to the logger:
+
+    ```typescript
+    import { Logger } from "@eyeseetea/d2-logger";
+
+    const logger = new Logger();
+
+    await logger.init({
+        type: "program",
+        debug: debug,
+        baseUrl: baseUrl,
+        auth: auth,
+        organisationUnitId: organisationUnitId,
+        programId: programId,
+        loggerDataElements: {
+            messageId: messageId,
+            messageTypeId: messageTypeId,
+        },
+    });
+
+    logger.debug("This is a Debug message");
+    logger.info("This is an Info message");
+    logger.success("This is a Success message");
+    logger.warn("This is a Warn message");
+    logger.error("This is an Error message");
+    ```
+
+    Notice:
+
+    - Please note that `auth` is not mandatory if it's used in the DHIS2 app instead of in a script.
+    - Please note that if the `organisationUnitId` is not added to the configuration, then the logs will be registered in the DHIS2 program in the global organization unit.
```

Comment:
Maybe it makes sense to a  unit of the program (the one with higher level?) instead of the global?

Reply:
For the moment, I've deleted this and also the default global org unit id, and I've made `organisationUnitId` mandatory. Maybe it is better to create the program, the data elements and also give access to the org unit. And if it's mandatory to pass it in the init, then it's safer that the program is in an org unit. Tomorrow we have GLASS follow-up and I'll comment this.